Makefile

```diff
@@ -156,6 +168,11 @@ prerelease: clean fmtcheck test cross ## Pre-release target to verify tests pass
 		exit 1; \
 	fi
 
+.PHONY: systemtray
+systemtray: clean fmtcheck test with_systemtray
+
+with_systemtray: $(BUILD_DIR)/darwin-amd64/systemtray/minishift $(BUILD_DIR)/windows-amd64/systemtray/minishift.exe
```

Comment:
This will not install the minishift binary in the go bin path. `make systemtray` or `make with_systemtray` should put the minishift binary in to go bin path similar to `make` otherwise it is in consistent.